LICENSE

```diff
@@ -298,6 +298,7 @@ License: https://www.apache.org/licenses/LICENSE-2.0
 This product includes code from Delta Lake.
 
 * AssignmentAlignmentSupport is an independent development but UpdateExpressionsSupport in Delta was used as a reference.
+* RoaringPositionBitmap is an independent development but RoaringBitmapArray in Delta was used as a reference.
```

Comment:
In some cases, it looks like the methods were directly translated. For example [`toPosition`](https://github.com/apache/iceberg/pull/11372/files#diff-42b322a03e4e0a70933627efc92563d6e47a7f707267a597fb1a5e94302d6517R293-R295) has a different name, but the same implementation and comment as [`composeFromHighLowBytes`](https://github.com/delta-io/delta/blob/master/spark/src/main/scala/org/apache/spark/sql/delta/deletionvectors/RoaringBitmapArray.scala#L498-L499). It may be better to mention this as a Java implementation based on Delta's Scala implementation.

Reply:
Will do.

Reply:
Updated.